src/app/blocks/adapter-info-block/adapter-info-block.component.ts

```diff
@@ -41,10 +47,19 @@ export class AdapterInfoBlockComponent extends BaseBlockComponent {
 
     this.hasError = false;
     this.isLoading = true;
+    
     this.localData['adapters'].get({ adapterName: this.adapterName })
-      .then(value => {
-        this.isLoading = false;
-        this.output.emit(value);
+      .then(adapterData => {
+        if (this.compileAdapter) {
+          this.adapterInstallService.compileAdapter(adapterData).then((compiledAdapter) => {
+            console.log(compiledAdapter);
+            this.isLoading = false;
+            this.output.emit(compiledAdapter);
+          });
+        } else {
+          this.isLoading = false;
+          this.output.emit(adapterData);
+        }
```

Comment:
If the compileAdapter flag is set, we use the AdapterInstallService to gather up all the adapter components, and return them all in a format compatible with export